release-notes.md

```diff
@@ -1,3 +1,12 @@
+- [v0.8.0](#v080)
+    - [Experimental Snapshot and Restore Support](#experimental-snapshot-and-restore-support)
+    - [Experimental ARM64 Support](#experimental-arm64-support)
+    - [Support for Using 5-level Paging in Guests](#support-for-using-5-level-paging-in-guests)
+    - [Virtio Device Interrupt Supression for Network Devices](#virtio-device-interrupt-supression-for-network-devices)
```

Comment:
Typo: `Supression` -> `Suppression` . There are also a few other same typos in this notes.

Reply:
Thanks.